lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -2266,60 +2259,7 @@ ParseResult FIRStmtParser::parseMemPort(MemDirAttr direction) {
       resultType, memory, indexExp, clock, direction, name, annotations);
   Value result = resultMemoryPort;
 
-  // TODO(firrtl scala bug): If the next operation is a skip, just eat it if it
-  // is at the same indent level as us.  This is a horrible hack on top of the
-  // following hack to work around a Scala bug.
-  auto nextIndent = getIndentation();
```

Comment:
I'd recommend splitting this change out to its own patch.  It is logically distinct from the rest of the patch, and we don't know if we need it :-)

Reply:
Removed! It makes way more sense to do this in a separate change